x-pack/solutions/security/plugins/security_solution/public/flyout/document_details/shared/hooks/use_event_details.ts

```diff
@@ -92,18 +95,33 @@ export const useEventDetails = ({
     pageName === SecurityPageName.detections
       ? SourcererScopeName.detections
       : SourcererScopeName.default;
+
   const sourcererDataView = useSourcererDataView(sourcererScope);
+
+  const oldRuntimeMappings: RunTimeMappings = sourcererDataView.sourcererDataView
+    .runtimeFieldMap as RunTimeMappings;
+  const oldBrowserFields = sourcererDataView.browserFields;
+
+  const newDataViewPickerEnabled = useIsExperimentalFeatureEnabled('newDataViewPickerEnabled');
+  const { dataView } = useDataView(sourcererScope);
+  const experimentalBrowserFields = useBrowserFields(sourcererScope);
+
+  const runtimeMappings = newDataViewPickerEnabled
+    ? (dataView?.getRuntimeMappings() as RunTimeMappings)
```

Comment:
Nit, but we shouldn't cast this. I don't even think we should have our own `RunTimeMappings` type as it's most likely out of sync with the default `estypes.MappingRuntimeFields`

Reply:
Created an issue for this https://github.com/elastic/security-team/issues/12440